pyproject.toml

```diff
@@ -18,14 +18,19 @@ maintainers = [
 ]
 
 dependencies = [ # Migration from requirements.txt
+    "torch<2.5.0",
```

Comment:
As discussed I don't believe this is needed after removing the pinned ocf_datapipes version

Reply:
Can try and remove it to double check the tests pass

Reply:
Nice good catch, it worked without the pin